Amplify/Categories/Auth/Models/AuthSession.swift

```diff
@@ -7,7 +7,11 @@
 
 import Foundation
 
+/// Defines the auth session behavior
 public protocol AuthSession {
 
+    /// Indicates whether a user is signed in or not
+    ///
+    /// `true` if a user is signed in
```

Comment:
As noted in discussion of a previous issue, "signed in" is ambiguous. What can we assert about the state of the system, and the behaviors of service queries, when this value is true or false?